### PR TITLE
Doc/security headers doc

### PR DESCRIPTION
**Pull request by loulou2u**
Documentation on setting up the properties for Web Security Headers

### CODE REVIEW

File: manuals/en/uportal5-manual/configure/security/headers.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+### X-Frame-Options
+
+Also known as antiClickJacking (https://tomcat.apache.org/tomcat-11.0-doc/config/filter.html), clickjacking is an attack that fools users into thinking they are clicking on one thing when they are actually clicking on another.
+
+The following antiClickJacking configurations are included in uPortal, and mimic the configuration options that are also available directly through the tomcat server config file.
+
+* antiClickJackingEnabled
+* antiClickJackingOptions
+* antiClickJackingUri
```
Comment (ChristianMurphy) [inline manuals/en/uportal5-manual/configure/security/headers.md:33]:
```suggestion
* `antiClickJackingEnabled`
* `antiClickJackingOptions`
* `antiClickJackingUri`
```

File: manuals/en/uportal5-manual/configure/security/headers.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+Locate the code block below and enable this feature by replacing "false" with "true". The antiClickJackingOptions available are "deny", "sameorigin", or "allow-from". If you select "allow-from", setup the URI for the antiClickJackingUri property. Learn more about which options are best suited for your needs at https://tomcat.apache.org/tomcat-11.0-doc/config/filter.html
```
Comment (ChristianMurphy) [inline manuals/en/uportal5-manual/configure/security/headers.md:35]:
```suggestion
Locate the code block below and enable this feature by replacing "false" with "true". The antiClickJackingOptions available are `deny`, `sameorigin`, or `allow-from`. If you select `allow-from`, setup the URI for the `antiClickJackingUri` property. Learn more about which options are best suited for your needs at https://tomcat.apache.org/tomcat-11.0-doc/config/filter.html
```

File: manuals/en/uportal5-manual/configure/security/headers.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+### Referrer-Policy
+
+Referrer-Policy controls how much referrer information should be included with requests.
+
+In the following code block located in the `security.properties` file, you can enable the header response by setting
+`sec.referrer.policy.enabled` to "true". In the `sec.referrer.policy` you may define your policy based on your preference by
+following the syntax defined at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy
+
+```plaintext
```
Comment (ChristianMurphy) [inline manuals/en/uportal5-manual/configure/security/headers.md:95]:
````suggestion
```properties
````

File: manuals/en/uportal5-manual/configure/security/headers.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+### Strict-Transport-Protocol
+
+Often referred to as HSTS, this response header informs the browser the site should only be accessed uding HTTPS,
+and that any attempts to access it using HTTP should be automatically be converted to HTTPS.
+
+There are a few directives that can be customized. You can refer to the documentation at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security for guidance. By default, the protocol is disabled and can be configured in the
+`security.properties` file. The following code block is the section you will customize in the properties file.
+
+```plaintext
```
Comment (ChristianMurphy) [inline manuals/en/uportal5-manual/configure/security/headers.md:67]:
````suggestion
```properties
````

File: manuals/en/uportal5-manual/configure/security/headers.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+ sec.anti.click.jacking.uri=
+```
+
+### Content-Security-Policy
+
+The Content-Security-Policy header allows you to restrict which resources can be loaded and the URLs that can they can be loaded from.
+
+By default, the content security policy is disabled in uPortal, but to enable it go to the `security.properties` file and switch the `sec.content.sec.policy.enabled` property to "true". To define the policy, see the documentation at https://content-security-policy.com/ to find the policy that works best for your purpose.
+
+```plaintext
```
Comment (ChristianMurphy) [inline manuals/en/uportal5-manual/configure/security/headers.md:52]:
````suggestion
```properties
````

File: manuals/en/uportal5-manual/configure/security/headers.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+The following antiClickJacking configurations are included in uPortal, and mimic the configuration options that are also available directly through the tomcat server config file.
+
+* antiClickJackingEnabled
+* antiClickJackingOptions
+* antiClickJackingUri
+
+Locate the code block below and enable this feature by replacing "false" with "true". The antiClickJackingOptions available are "deny", "sameorigin", or "allow-from". If you select "allow-from", setup the URI for the antiClickJackingUri property. Learn more about which options are best suited for your needs at https://tomcat.apache.org/tomcat-11.0-doc/config/filter.html
+
+```plaintext
```
Comment (ChristianMurphy) [inline manuals/en/uportal5-manual/configure/security/headers.md:37]:
````suggestion
```properties
````

**Review by ChristianMurphy — APPROVED** [review]
Thanks @loulou2u!

File: manuals/en/uportal5-manual/configure/security/headers.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+* X-Frame-Options (antiClickJacking)
+* Content-Security-Policy
+* Strict-Transport-Protocol
+* X-Content-Type-Option
+* Referrer-Policy
+
+## Where to Configure
+
+To customize the security header response options, go to the following file in uPortal:
```
Comment (bjagg) [inline manuals/en/uportal5-manual/configure/security/headers.md:19]:
```suggestion
To customize the security header response options, add the desired security header keys and their preferred values to `etc/portal/uPortal.properties` in uPortal-start repo, copy this file to $PORTAL_HOME and restart uPortal.

To review the security headers and their defaults, go to the following file in uPortal:
```

**Review by bjagg — CHANGES_REQUESTED** [review]

**Review by bjagg — APPROVED** [review]
🥇

**Review by ChristianMurphy — APPROVED** [review]

**Pull request merged**